src/shopping-cart/actions/add-item-to-shopping-cart.js

```diff
@@ -0,0 +1,7 @@
+export const ADD_ITEM_TO_SHOPPING_CART = 'ADD_ITEM_TO_SHOPPING_CART'
+export const REMOVE_ITEM_FROM_SHOPPING_CART = 'REMOVE_ITEM_FROM_SHOPPING_CART'
+
+
+export const addItemToShoppingCart = item => ({type: ADD_ITEM_TO_SHOPPING_CART, payload: item})
+export const removeItemFromShoppingCart = remainingItems => 
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/109503901-85530300-7aa3-11eb-92ff-6a3ed38e92ba.png)

Let's move the `remove` action to a different file.

Reply:
![image](https://user-images.githubusercontent.com/23569938/109504082-c9de9e80-7aa3-11eb-85fb-f79dba9e2158.png)
Also, since we pass the items to remove to the action, not the remaining items, maybe rename this parameter?

Reply:
Also, since we pass a multitude of items to these afore-mentioned actions, maybe rename them like this: 
* addItemToShoppingCart -> addItem**s**
* removeItemFromShoppingCart -> removeItem**s**

We can also omit the `ShoppingCart` part since we're already in the context of the `shopping-cart` directory:
![image](https://user-images.githubusercontent.com/23569938/109504661-7882df00-7aa4-11eb-9b42-5b1f95e960da.png)

If you really want more context, you can import them using an alias like this:
`import { addItems as addItemsToShoppingCart } from 'whatever'`

Reply:
Done